mmdeploy/codebase/mmdet3d/deploy/voxel_detection.py

```diff
@@ -165,10 +225,7 @@ def run_inference(model: nn.Module,
         Returns:
             list: The predictions of model inference.
         """
-        result = model(
-            return_loss=False,
-            points=model_inputs['points'],
-            img_metas=model_inputs['img_metas'])
+        result = model(inputs=model_inputs['inputs'])
```

Comment:
this func should be removed since it not used.

Reply:
done.